seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java

```diff
@@ -28,17 +28,17 @@
 public class HiveJdbcDataSourceFactory implements DataSourceFactory {
     @Override
     public String factoryIdentifier() {
-        return "Hive-JDBC";
+        return "JDBC-Hive";
```

Comment:
@arshadmohammad Can `JDBC-Hive` be extracted as a common variable parameter

Reply:
sure, Thanks @wuchunfu  for the review.

Reply:
The review comment has been addressed